backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/shipment/M_InOut_StepDef.java

```diff
@@ -620,6 +621,22 @@ public void shipment_action_expecting_error(@NonNull final String model_UNUSED,
 		assertThat(errorThrown).isTrue();
 	}
 
+	@And("^after not more than (.*)s, locate reversal M_InOut$")
+	public void find_reversal_M_InOut(final int timeoutSec, @NonNull final DataTable dataTable) throws InterruptedException
+	{
+		for (final Map<String, String> row : dataTable.asMaps())
+		{
+			final I_M_InOut reversalInOut = StepDefUtil.tryAndWaitForItem(timeoutSec, 500, () -> load_reversal_InOut(row));
+
+			final SoftAssertions softly = new SoftAssertions();
+			softly.assertThat(reversalInOut).isNotNull();
+			softly.assertAll();
+
+			final String reversalInOutIdentifier = DataTableUtil.extractStringForColumnName(row, COLUMNNAME_M_InOut_ID + "." + TABLECOLUMN_IDENTIFIER);
+			shipmentTable.putOrReplace(reversalInOutIdentifier, reversalInOut);
+		}
+	}
```

Comment:
Why not calling softly.assertAll() out of loop?
In the that's the purpose of using soft assertions.